docs/guides/unity-catalog.md

```diff
@@ -36,47 +156,132 @@ data "aws_iam_policy_document" "passrole_for_uc" {
+    Id      = "${local.prefix}-databricks-sample-data"
+    Statement = [
+      {
+        "Action": [
           "s3:GetObject",
-          "s3:GetBucketLocation",
-          "s3:DeleteObject",
-        ]
-        Effect = "Allow"
-        Resource = [
-          aws_s3_bucket.metastore.arn,
-          "${aws_s3_bucket.metastore.arn}/*"
-        ]
-      }]
-    })
-  }
+          "s3:GetObjectVersion",
+          "s3:ListBucket",
+          "s3:GetBucketLocation"
+        ],
+        "Resource": [
+          "arn:aws:s3:::databricks-datasets-oregon/*",
+          "arn:aws:s3:::databricks-datasets-oregon"
+
+        ],
+        "Effect": "Allow"
+      }
+    ]
+  })
+  tags = merge(local.tags, {
+    Name = "${local.prefix}-unity-catalog IAM policy"
+  })
 }
 
+resource "aws_iam_role" "metastore_data_access" {
+  name               = "${local.prefix}-uc-access"
+  assume_role_policy = data.aws_iam_policy_document.passrole_for_uc.json
+  managed_policy_arns = [aws_iam_policy.unity_metastore.arn, aws_iam_policy.sample_data.arn]
+  tags = merge(local.tags, {
+    Name = "${local.prefix}-unity-catalog IAM role"
+  })  
+}
+```
+
+## Create users and groups
+
+A Unity Catalog metastore can be shared across multiple Databricks workspaces. To enable this, Databricks must have a consistent view of users and groups across all workspaces, and has introduced features within the account console to manage this. Users and groups that wish to use Unity Catalog must be created as account level identities
+
+---
+**Note**
+
+Databricks does not allow a single user to be added to more than one account. You will receive the error `User already exists in another account`
+
+---
+
+```hcl
+resource "databricks_user" "unity_users" {
+  provider  = databricks.mws
+  for_each  = toset(concat(var.databricks_users, var.databricks_unity_admins))
+  user_name = each.key
+  force     = true
+}
+
+resource "databricks_group" "admin_group" {
+  provider     = databricks.mws
+  display_name = var.unity_admin_group
+}
+
+resource "databricks_group_member" "admin_group_member" {
+  provider  = databricks.mws
+  for_each  = toset(var.databricks_unity_admins)
+  group_id  = databricks_group.admin_group.id
+  member_id = databricks_user.unity_users[each.value].id
+}
+
+resource "databricks_user_role" "my_user_account_admin" {
+  provider = databricks.mws
+  for_each = toset(var.databricks_unity_admins)
+  user_id  = databricks_user.unity_users[each.value].id
+  role     = "account_admin"
+}
+```
+## Create a Unity Catalog metastore and link it to workspaces
+
+A metastore is the top level container for data in Unity Catalog. A single metastore can be shared across Databricks workspaces, and each linked workspace has a consistent view of the data and a single set of access policies. It is only recommended to have multiple metastores when organizations wish to have hard isolation boundaries between data (note that data cannot be easily joined/queried across metastores).
```

Comment:
every time we mention an entity that has TF resource, we should make a link, e.g. `[databricks_metastore](../resources/metastore.md)` instead of `metastore`.

Reply:
done